src/components/Reports/SalaryReport/SalaryReportTable.tsx

```diff
@@ -1,10 +1,214 @@
-import React from 'react';
-import { Box } from '@material-ui/core';
+/* eslint-disable import/no-unresolved */
+import React, { useMemo, useState } from 'react';
+import clsx from 'clsx';
+import {
+  Box,
+  Button,
+  ButtonGroup,
+  Grid,
+  styled,
+  SvgIcon,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+  makeStyles,
+} from '@material-ui/core';
+import GetAppIcon from '@material-ui/icons/GetApp';
+import CodeIcon from '@material-ui/icons/Code';
+import PrintIcon from '@material-ui/icons/Print';
+import { CSVLink } from 'react-csv';
+import { DateTime } from 'luxon';
+import { useTranslation } from 'react-i18next';
+import {
+  FourteenMonthReportContact,
+  FourteenMonthReportCurrencyType,
+  // eslint-disable-next-line import/extensions
+} from 'graphql/types.generated';
+// eslint-disable-next-line import/extensions
+import { useFourteenMonthReportQuery } from 'pages/accountLists/[accountListId]/reports/graphql/GetReportFourteenMonth.generated';
+import Loading from 'src/components/Loading';
+import { Notification } from 'src/components/Notification/Notification';
+import { EmptyReport } from 'src/components/Reports/EmptyReport/EmptyReport';
 
 interface Props {
+  className?: string;
   accountListId: string;
+  title: string;
 }
 
-export const SalaryReportTable: React.FC<Props> = () => {
-  return <Box />;
+const useStyles = makeStyles(() => ({
+  root: {},
+  downloadCsv: {
+    color: 'inherit',
+    textDecoration: 'none',
+  },
+}));
+
+const DownloadCsvLink = styled(CSVLink)(({}) => ({
+  color: 'inherit',
+  textDecoration: 'none',
+}));
+
+export const SalaryReportTable: React.FC<Props> = ({
+  className,
+  accountListId,
+  title,
+  ...rest
+}) => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+  const [contacts, setContacts] = useState<FourteenMonthReportContact[]>([]);
```

Comment:
Using this type works. It's kinda verbose but 🤷‍♂️ 
```suggestion
  const [contacts, setContacts] = useState<
    FourteenMonthReportQuery['fourteenMonthReport']['currencyGroups'][0]['contacts'][]
  >([]);
```